k8s/jobs/django-rebuild-search-job.yml

```
@@ -0,0 +1,18 @@
+  name: django-rebuild-search-job
+  namespace: default
+spec:
+  backoffLimit: 1
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: django-rebuild-search
+          image: registry.digitalocean.com/mindjunkies/django-mindjunkies-web:latest
+          imagePullPolicy: Always
+          command: [ "uv", "run", "python", "manage.py", "search_index", "--rebuild", "--noinput" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)